scripts/bump-edge.ts

```diff
@@ -117,6 +119,11 @@ async function main() {
       `${pkg.data.version}-${date}.${commit}`
     );
     workspace.rename(pkg.data.name, pkg.data.name + "-edge");
+    pkg.updateDeps((dep) => {
+      if (nightlyPackages[dep.name]) {
+        dep.range = "npm:" + nightlyPackages[dep.name] + "@latest";
```

Comment:
see my comment at https://github.com/nuxt/nuxt/pull/22593#discussion_r1291241882.

I'm not sure `latest` is safe here. I'd recommend a caret + version hash.

Reply:
Also see my comment there! I think it should be safer than pinned version to avoid hoisting issues on edge.

Reply:
Reposting from https://github.com/nuxt/nuxt/pull/22593#discussion_r1291262322, caret + hash is respected by package manager in a way that it won't look for any newer version + (sadly) `h3-nightly@^1` doesn't works with packages published with `-` version.